SWIG/indexes.i

```diff
@@ -432,6 +432,11 @@ class Name##Ptr : public Base##Ptr {
 };
 %enddef
 
+%inline %{
+    SwapIndexPtr as_swapIndex(const InterestRateIndexPtr& index) {
```

Comment:
I'd go for `as_swap_index` as we did for the cashflow casts (but unfortunately not for `as_iborindex`...)

Reply:
Fine for me, will rename accordingly.